modules/persistent-storage-csi-ebs-operator-install.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-ebs.adoc
+
+[id="persistent-storage-csi-ebs-operator-install_{context}"]
+= Installing the AWS Elastic Block Store CSI Operator
+
+The AWS Elastic Block Store (EBS) Container Storage Interface (CSI) Operator enables the replacement of the existing AWS EBS in-tree storage plug-in.
```

Comment:
nit: we call it _AWS EBS CSI driver operator_, so we're missing a _CSI_ :)

Please ignore if that's how we want to refer to it in the docs.

Reply:
Hmm, I'm not sure where it's missing _CSI_. I don't have CSI as part of "AWS EBS in-tree storage plug-in" because it's in-tree vs. CSI. Sorry if I'm not understanding correctly here.

Reply:
Oh, sorry, I meant _Driver_ instead of _CSI_(I updated the comment) :+1:

Reply:
Oh, I see. That';s a good question. I was using the name based on the [STOR-312](https://issues.redhat.com/browse/STOR-312) epic title. I can't test on 4.5 yet, do you know what it will be called in OperatorHub?

Reply:
I noticed it's called "AWS EBS CSI Operator" in OperatorHub, but that's a bug. I opened a ticket to fix that (https://bugzilla.redhat.com/show_bug.cgi?id=1842402).

Once the bug is fixed, it's going to be called "AWS EBS CSI Driver Operator".

Reply:
Ah, OK. Got it. I'll need to update the doc to reflect this name, thanks!